static/examples/cstg-prebid-example/README.md

```diff
@@ -1,7 +1,9 @@
-# Prebid.js UID2 with client-side token generation
+# Example Prebid.js UID2 Integration
 
 ## Viewing live site
 
+This example demonstrates the Prebid.js Express Integration guide [here](https://unifiedid.com/docs/guides/integration-prebid). 
```

Comment:
Best not to have [here] links. Suggest:
This example demonstrates the approach detailed in the [Prebid.js Express Integration Guide](https://unifiedid.com/docs/guides/integration-prebid).

Reply:
fixed